src/hooks/customs/useAuth.tsx

```
@@ -0,0 +1,66 @@
+import { useEffect } from 'react';
+import { usePathname, useRouter } from 'next/navigation';
+import { useAuthStore } from '@/store/useAuthStore';
+import { jwtDecode } from 'jwt-decode';
+
+//함수: 토큰 디코딩하여 시간 만료되었는지 유효성 검증
+const isTokenExpired = (token: string): boolean => {
+	try {
+		const decoded: any = jwtDecode(token); // 토큰 디코딩
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)